src/brokers/broker-details/components/yaml/Yaml.container.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, Suspense } from 'react';
+import { Loading } from '@app/shared-components/Loading/Loading';
+import { ResourceYAMLEditor } from '@openshift-console/dynamic-plugin-sdk';
+import YAML from 'yaml';
+import { Alert, AlertVariant, Button, Page } from '@patternfly/react-core';
+import { useTranslation } from '@app/i18n/i18n';
+import { useNavigate, useParams } from 'react-router-dom-v5-compat';
+
+type YamlViewProps = {
+  brokerCr: object;
+};
+
+const YamlContainer: FC<YamlViewProps> = ({ brokerCr }) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const { ns: namespace, name } = useParams<{ ns?: string; name?: string }>();
+  const onClickEditYaml = () => {
+    const currentPath = window.location.pathname;
+    navigate(
+      `/k8s/ns/${namespace}/edit-broker/${name}?returnUrl=${encodeURIComponent(
+        currentPath,
+      )}`,
+    );
+  };
+
+  return (
+    <Page>
```

Comment:
This can't be a `<Page>`, because it's inside a `<Page>` already, it can only be a `<PageSection>`